ruby/rage/Gemfile

```
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+
+gem "rage-rb", "~> 1.3"
```

```suggestion
gem "rage-rb", "~> 1.3.0"
```


To make sure 1.3.x is used